src/index.ts

```diff
@@ -18,7 +21,11 @@ export const logger = () =>
             logStr.push(methodString(request.method))
 
             logStr.push(new URL(request.url).pathname)
-            const beforeTime: [number, number] = (store as any).beforeTime;
+            const beforeTime: bigint = (store as any).beforeTime;
+
+
+            console.log(`Before Time from After Handle - ${beforeTime}`)
```

Comment:
Remove this. No logging except for the requests, please.

Reply:
I forgot to remove the log